### PR TITLE
Feature: Trendline added to RevocationsOverTime chart.

### LINKED ISSUE #337: [Lantern] For the line plotting number of revocations over time, add a trend line
**Is your feature request related to a problem? Please describe.**
We do not currently show the trend in revocation for the time period shown. We would like a way for users to easily see if revocations are generally trending up or down for the time period selected.

**Describe the solution you'd like**
For the time period selected, add a trend line running across the full horizontal length of the chart, similar to the charts with trend lines on the existing ND dash.

image

**Describe alternatives you've considered**
N/A

**Additional context**
TODO: There is an open followup item to discuss what a "significance test" would mean for this and decide whether one is necessary (this could add a non-trivial amount of work, pending the outcome of the discussion).

### PR DESCRIPTION
**Pull request by Wmaileq**
## Description of the change

- Trendline added

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #337

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/utils/charts/__tests__/trendline.test.js [hunk 1]
```diff
@@ -14,20 +14,20 @@
 // You should have received a copy of the GNU General Public License
```
Comment (jessex) [inline src/utils/charts/__tests__/trendline.test.js:14]: It looks like we made a bunch of lint changes in some of these files? Let's make sure they are all whitelisted in `.eslintignore`.

**Review by jessex — CHANGES_REQUESTED** [review]
Looks good, @Wmaileq! One request.

File: src/utils/charts/__tests__/trendline.test.js (hunk 1 above)
Comment (Wmaileq) [reply src/utils/charts/__tests__/trendline.test.js:14]: Added to whitelist and checked with eslint, lets wait for successful travis build and merge.

Comment (jessex) [reply src/utils/charts/__tests__/trendline.test.js:14]: Thanks, @Wmaileq! Just getting permission from product to merge this one before we hit the green button.

**Review by jessex — APPROVED** [review]
🚀

**Pull request merged**